xml_inscriptions/bib_only/MA.Bos.MFA.G.21.1843.xml

```diff
@@ -35,24 +35,95 @@
+                           <dimensions type="surface" unit="cm">
+                              <height>15</height>
+                              <width>15</width>
+                              <depth/>
+                           </dimensions>
+                        </support>
+                        <condition ana="#complete.intact">
+                           <p/>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="1">
+                           <p> on rim of bowl</p>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  
+                  <handDesc hands="1">
+                     <handNote ana="#etched"> </handNote>
+                  </handDesc>
+                  
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               
+               <!--  *************************************  <history>  *********************************  -->
+               
+               <history>
+                  
+                  <summary>
+                     <rs/>  <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                  </summary>
+                  
+                  <origin>
+                     <date evidence="#history" notBefore="-0626" notAfter="-0624">about 625
+                        B.C.</date>
+                     <placeName ref="europe.greece">Corinth</placeName>
+                  </origin>
+                  <provenance type="transferred" subtype="purchased">
+                     <p> purchased by <orgName>MFA</orgName>, <date when="1921">September 1, 1921</date>, for $8,000.00</p>
+                  </provenance>
+                  <provenance type="found" subtype="discovered">
+                     <p> said to come from <placeName>Olympia</placeName>, discovered five years ago east of the Altis
+                        between the stadium and the river Alpheios, in the bank of a small torrent
+                        formed by winter rains</p>
+                  </provenance>
+                  <provenance type="observed" subtype="reported">
+                     <p> By <date when="1921">1921</date>: with <persName>Mr. D. K. Tseklenis</persName> and <persName>Mr. Stratos</persName>, 34 Bromfield Street, <placeName>Boston</placeName></p>
```

Comment:
Please try to remember to add `@role` attributes on `<persName>` and `<orgName>`, where possible to discern their roles